packages/carbon-web-components/package.json

```diff
@@ -192,7 +192,7 @@
     "rollup-plugin-multi-input": "^1.3.1",
     "rollup-plugin-terser": "^7.0.2",
     "rtlcss": "^3.0.0",
-    "sass": "~1.57.0",
+    "sass": "~1.40.0",
```

Comment:
@annawen1 Looks like @ariellalgilmore already updated the version [here](https://github.com/emyarod/carbon-for-ibm-dotcom/pull/3/files#diff-065807478c266e15c7d9c6cf219d38a8ba9035da6c18c46e7a0f21f44f651ffcR195).